src/wp-includes/ms-site.php

```diff
@@ -498,7 +498,16 @@ function wp_normalize_site_data( $data ) {
 	// Sanitize domain if passed.
 	if ( array_key_exists( 'domain', $data ) ) {
 		$data['domain'] = trim( $data['domain'] );
+		$has_ports = strstr( $data['domain'], ':' );
+
+		$data['domain'] = preg_replace( '|:\d+$|', '', $data['domain'] ); // Remove ports.
```

Comment:
```suggestion
		// Remove ports.
		$data['domain'] = preg_replace( '|:\d+$|', '', $data['domain'] );
```

Smaller screens require horizontal scrolling to see this comment. Let's move it above the line to make things a little easier for those browsing the code.